automation-roles/50-install-cloud-pak/cp4wa/cp4wa-cluster/templates/cp4wa-catalog-sources.j2

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
For CP4D and CP4I we have stepped away from using the ibm-operator-catalog and plan to only support installation through case files to have a specific release. I would recommend to do the same for CP4WA.

Reply:
Can I create an issue for this and add to the next iteration? Are you saying we should remove the option for ibm-operator-catalog as well?